osmhelper/osm_helper.py

```diff
@@ -69,25 +84,19 @@ def read_stops_from_file():
             return {}
 
     @staticmethod
-    def get_routes(route_type, bbox, refresh=False):
+    def get_routes(bbox, tags, refresh=False):
         if refresh:
             print "Start with fresh routes"
             routes = {}
         else:
             routes = OsmHelper.read_routes_from_file()
 
         # get the master routes first
-        route_masters = OsmHelper.get_route_masters(route_type, bbox, refresh)
+        route_masters = OsmHelper.get_route_masters(bbox, tags, refresh)
 
         api = overpy.Overpass()
-
-        result = api.query("""
-        <query type="relation">
-            <has-kv k="route" v="%s"/>
-            <bbox-query e="%s" n="%s" s="%s" w="%s"/>
-        </query>
-        <print/>
-        """ % (route_type, str(bbox["e"]), str(bbox["n"]), str(bbox["s"]), str(bbox["w"])))
+        query_str = 'relation(%s)%s;(._;);out body;' % (bbox, tags)
```

Comment:
Ok, I'll add to the config file the tag (osm k-v) "route" = "bus".

~~~
  "tags": {
            "route": "bus"
        }
~~~

I'm participating in a Informatics Conference in Chile during this week. I have good Internet access some hours a day but still haven't been able to test all code. Yesterday I tried but I got the Too many requests exception.  

I know is not Ok, but again I'll upload the changes just in case. I'll keep trying to test the code. For sure I could test it next week.